gensim/parsing/preprocessing.py

```diff
@@ -96,6 +96,26 @@ def strip_short(s, minsize=3):
 
 
 def strip_numeric(s):
+
+    """Takes string and removes digits from it.
```

Comment:
Coding style: docstring in Python should be in imperative mode: "Do X", not "Does X".